addons/web/static/src/legacy/js/widgets/data_export.js

```diff
@@ -397,10 +397,17 @@ var DataExport = Dialog.extend({
             self._onShowData(records);
             self.$('.o_fields_list').empty();
 
+            let forceDefaultExportFields = _.chain(records)
```

Comment:
Are the chain + value necessary ? Can't you just filter + map ?

Reply:
Yes using the "new" lambda syntax, it can be nicely expressed in one line. Thanks
(if not in one line, I find the chain syntax more readable and I thought it was faster (using maybe iterator, ...) but it seems not the case)